pkgs/applications/science/math/pagsuite/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchzip
+, cmake
+, gmp
+, scalp
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pagsuite";
+  version = "1.80";
+
+  src = fetchzip {
```

Comment:
Is there a reason to use `fetchzip` here and not `fetchFromGitLab`?

Reply:
The version was released before the transition to Git.

Reply:
I can't find the release online https://gitlab.com/kumm/pagsuite/-/releases